src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/GroupsController.kt

```diff
@@ -116,6 +120,48 @@ class GroupsController(
   ) {
     groupsService.updateChildGroup(group, groupAmendment)
   }
+
+  @PostMapping("/groups")
+  @PreAuthorize("hasRole('ROLE_MAINTAIN_OAUTH_USERS')")
+  @Operation(
+    summary = "Create group.",
+    description = "Create a Group"
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        responseCode = "200",
+        description = "OK"
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorized.",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class)
+          )
+        ]
+      ),
+      ApiResponse(
+        responseCode = "409",
+        description = "Group already exists.",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class)
+          )
+        ]
+      )
+    ]
+  )
+  fun createGroup(
+    @Parameter(description = "Details of the group to be created.", required = true)
```

Comment:
Not sure if @Parameter is the correct annotation here

Reply:
Updated